from-repo/c-cpp/README.md

```diff
@@ -0,0 +1,14 @@
+# Auto-build oss-fuzz projects from a GitHub repo URL
+
+
+To run:
+
+```sh
+# prepare
+export OPENAI_API_KEY=your-api-key
```

Comment:
Does this work with vertex ai as well? if so, we should add some setup steps for that also.

Reply:
Not yet. I need to adjust a few things because then harness generation happens inside a Docker container and there's no API key. I don't think this is a blocker as such but need to find out how to easily handle vertex calls within a container environment (i.e. no gcloud login).

Is this a blocker for landing?

Reply:
not a blocker for this one, but something we definitely need to address in a subsequent PR.

Reply:
can you add a TODO? one way is to just use a Google cloud service account, and set GOOGLE_APPLICATION_CREDENTIALS: https://cloud.google.com/docs/authentication/application-default-credentials#GAC

Reply:
I'll see if I can get this in tomorrow before landing this

Reply:
I've added this now and also adjusted the README